src/Controls/src/Core/Compatibility/Handlers/Shell/Android/ShellFlyoutTemplatedContentRenderer.cs

```diff
@@ -259,15 +259,14 @@ protected virtual void UpdateFlyoutFooter()
 
 			UpdateFooterLayout();
 			UpdateContentLayout();
-			UpdateContentBottomMargin();
 		}
 
 		void UpdateFooterLayout()
 		{
-			if (_footerView != null)
-			{
-				_footerView.LayoutView(0, 0, _rootView.LayoutParameters.Width, MeasureSpecMode.Unspecified.MakeMeasureSpec(0));
-			}
+			_ = _footerView?
+					.Measure(MeasureSpecMode.Exactly
+					.MakeMeasureSpec(_rootView.LayoutParameters.Width), MeasureSpecMode.Unspecified.MakeMeasureSpec(0), null, null);
```

Comment:
Why is there a discard `_` here? Can we not just use the method normally?

Reply:
It's informational. It says "yea I know this thing returns a size but I'm intentionally ignoring it."

It seems helpful for future folks :-)